### PR TITLE
[onert] Resolve svace for UNINIT and DIVIDE_BY_ZERO

### PR DESCRIPTION
**Pull request by YiHyunJin**
Initialize uninitialized variables.
Assert if the value to divide by is zero.

Signed-off-by: hj0412-yi <hj0412.yi@samsung.com>

### CODE REVIEW

File: runtime/onert/backend/gpu_cl/open_cl/ClCommandQueue.cc [hunk 1]
```diff
@@ -268,6 +268,7 @@ ProfilingCommandQueue::GetBestWorkGroupIndex(const CLKernel &kernel, const Devic
         average_samples_count++;
       }
     }
+    assert(average_samples_count > 0);
```
Comment (seanshpark) [inline runtime/onert/backend/gpu_cl/open_cl/ClCommandQueue.cc:271]: `assert` will be ignored in release build. is this change only for debug?

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Comment by YiHyunJin** [comment]
Related : #7111

**Comment by chunseoklee** [comment]
@YiHyunJin Aside from this PR, could you please use coverity if it is coveirty defect ? AFAIK, svace issues are all resolved for now.

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**